### ISSUE TITLE
[Documentation / Clarification] 

### ISSUE THREAD

**Issue by rubyFeedback** [ostreedev/ostree#2141]
Hey ostree devs,

There is another project called ostree at gnome:

http://ftp.gnome.org/pub/gnome/sources/ostree/

This one has not been updated in ~6 years or so.

I came to libostree here via flatpak because flatpak said that I am missing
a dependency. I finally found out that it referred to libostree, rather than
that gnome ostree, so things are compiling now.

Still, I came to perhaps ask if it were possible if you guys could add a 
very short subsection about that gnome ostree project? Is it in any
way related to libostree? I mention this specifically because perhaps 
a few other people may also run into that problem, so a brief
entry in the README.md may perhaps be useful. At any rate please
feel free to close this issue request at any moment in time! Thanks.

**Comment by cgwalters** [comment]
Yeah, we transitioned to Github primarily, see https://mail.gnome.org/archives/ostree-list/2016-March/msg00005.html
